docs/4.4/api-reference.md

```diff
@@ -221,7 +373,7 @@ if err := client.DeleteToken(tokenString); err != nil {
 
 The root cluster in a [trusted cluster](admin-guide.md#adding-nodes-to-the-cluster) can add/update cluster labels on leaf clusters.
 
-**Create a leaf cluster token with Labels**
+### Create a leaf cluster token with Labels
 
 Leaf clusters will inherit the labels of the [join token](trustedclusters.md#join-tokens) used to add them to the trusted cluster. This is the preferred method of managing cluster tokens.
```

Comment:
I'm not sure that this sentence makes sense currently - join tokens don't have labels set on them AFAIK?

Reply:
They do as of this recent pr #4630